changelog.md

```diff
@@ -1,19 +1,23 @@
 # Changelog
 
-## v0.6.0 | t.b.d.
+## v0.6.0 | 2020-08-18
+
+* Plot and return images and timestamps for scans using physical coordinate system rather than fast and slow scanning axis. See docs tutorial section on `Confocal images` and `Kymographs` on how to use the API. **Note that this is a breaking change for scans with the fast axis in y direction!**
+* `Scan.pixels_per_line` and `Kymo.pixels_per_line` return the number of pixels along the fast axis now. **Note that this is a breaking change for scans/kymos with the fast axis in y direction!**
+* Verify starting timestamp when reconstructing `Kymo` or `Scan`. In those cases, scans cannot be reliably reconstructed and an error is thrown. For kymos, the first (partial) line is omitted and a warning is issued. **Note that this is a breaking change for scans and kymos where the scan/kymo was initiated before the exported time window!**
```

Comment:
I'm not sure that this is really a breaking change. The old scans were clearly wrong. Unless I'm missing something, I'd remove the breaking notice here and just mention that before this fix some partial scans may be been wrong.

Reply:
Yes, good point. I feel the warning could still use a bold though in case someone suddenly gets exceptions and wants to know why :)